lib/Server.js

```diff
@@ -235,7 +235,10 @@ function Server(compiler, options) {
 		historyApiFallback: function() {
 			if(options.historyApiFallback) {
 				// Fall back to /index.html if nothing else matches.
-				app.use(historyApiFallback(typeof options.historyApiFallback === "object" ? options.historyApiFallback : null));
+				app.use(historyApiFallback(
```

Comment:
Can you move `historyApiFallback` to the line under it? That would make clear that `express.static` is passed as second param to `app.use`